egs/wsj/s5/steps/data/gen_wav_sample.py

```diff
@@ -0,0 +1,103 @@
+#encoding:utf8
```

Comment:
Dan is encouraging using python3 for all new python scripts... so you might want to put #!/usr/bin/env python3 at the top of the script.

Reply:
Also, can you add some comment at the top of the file explaining what it does?